### ISSUE TITLE
Cannot read property 'apply' of undefined in version 2.3.3

### ISSUE THREAD

**Issue by pfefferniels** [rism-ch/verovio#1247]
Setting up an empty node.js project with only those two lines

```
const verovio = require('verovio'),
      vrvToolkit = new verovio.toolkit();
```
and verovio 2.3.3 installed via npm, I'm getting this (tested on different machines):
```
TypeError: Cannot read property 'apply' of undefined
    at Object.Module._vrvToolkit_constructor [as constructor] (.../test/node_modules/verovio/index.js:73:100412)
    at new verovio.toolkit (.../test/node_modules/verovio/index.js:157:32)
    at Object.<anonymous> (.../test/server.js:2:20)
    at Module._compile (internal/modules/cjs/loader.js:1139:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1159:10)
    at Module.load (internal/modules/cjs/loader.js:988:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
```
I suppose this is related to #1199, but version 2.3.3 didn't seem to fix it for me. Up to version 2.2.1, everything works just fine.

**Comment by lpugin** [comment]
This was due to asynchronous loading not handled properly. This should now be fixed with 2.3.4-1. See the updated documentation on https://www.npmjs.com/package/verovio

**Comment by lpugin** [comment]
See https://github.com/rism-ch/verovio/issues/1247#issuecomment-570592845
